pyppeteer/task_queue.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Task Queue module
+
+puppeteer equivalent: lib/TaskQueue.js
+"""
+import asyncio
+from typing import Awaitable
+
+
+class TaskQueue:
+    def __init__(self):
+        self._last_future = asyncio.Future()
+        self._last_future.set_result(None)
+
+    async def postTask(self, task: Awaitable):
+        result = await self._last_future
```

Comment:
I'm not sure if this is an exact translation from the js

Reply:
Unfortunately, I don't think I fully understand the purpose of this javascript code. Maybe this could be of use: https://docs.python.org/3/library/asyncio-queue.html

Check out the commit message for adding the TaskQueue class: https://github.com/puppeteer/puppeteer/commit/21af495b653a35e2a783b4d10fcb8697a0de85af